modules/navier_stokes/test/tests/finite_volume/limiters/lid-driven-segregated/tests

```diff
@@ -12,10 +12,12 @@
       heavy = true
       recover = false # we don't support recovery for SIMPLE yet
       max_threads = 1 # see libmesh issue #3808
```

Comment:
all these segregated tests should use continue_on_max_its

Reply:
This will result trivial regolding as well because now the output will contain two time steps.

Reply:
as it should. This was not trivial, the previous gold file was missing the solution